workspaces/ocm/plugins/ocm-common/src/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from './types';
+export * from './permissions';
+
+export const ANNOTATION_CLUSTER_ID = 'backstage-community.io/ocm-cluster-id';
+export const ANNOTATION_PROVIDER_ID = 'backstage-community.io/ocm-provider-id';
```

Comment:
Hmm, actually now that I think about it this might not be safe because users might already have entities using the old ANNOTATIONS and I'm not sure if us switching to a new annotation would cause them issues

Reply:
Do you think we should keep the janus annotations for now? And then later open a followup PR if it needs to be changed?